libnmstate/varlink/__init__.py

```diff
@@ -0,0 +1,257 @@
+#
+# Copyright (c) 2018-2020 Red Hat, Inc.
```

Comment:
This should be `Copyright (c) 2020 Red Hat, Inc.` because it is a new file.

Reply:
I'll fix it.